.github/workflows/test.yml

```diff
@@ -63,17 +67,19 @@ jobs:
 
   build-debian-package:
     runs-on: ubuntu-latest
+    container: debian:10
 
     steps:
       - uses: actions/checkout@v3
         with:
           path: clone
       - name: Install dependencies
         run: |
-          sudo apt-get update
-          sudo apt-get install build-essential dpkg-dev
+          apt-get update
+          apt-get install -y build-essential dpkg-dev ca-certificates sudo curl
```

Comment:
Do you remember what you needed these for? Especially `sudo` is a little surprising in a container.

Reply:
I believe `actions-rust-lang/setup-rust-toolchain` wanted them, but I can check in a bit.